mkosi/resources/mkosi.md

```diff
@@ -625,6 +625,12 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     `/usr/lib/os-release` or similar, in particular the `IMAGE_VERSION=`
     field of it).
 
+`ImageVersionFile=`, `--image-version-file=`
+:   Takes a path to a file to read the `ImageVersion=` from if not
+    explicitly set. The file will also be used when running `mkosi bump`
+    or when building with the `--auto-bump` option enabled. This value
+    default to `mkosi.version` if it is found in the config directory.
```

Comment:
I'm not a big fan of a separate setting for this. I'd prefer to just make `mkosi.version` the default path for `ImageVersion=`

Reply:
Dropped the second commit.